integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ReceiptPolling.java

```diff
@@ -29,11 +28,15 @@
     @Autowired
     private MessageReceiptRepository messageReceiptRepository;
 
+    // TODO: fjernes etter test!
     @PostConstruct
     private void addTestData() {
-        // TODO: fjernes etter test!
-        messageReceiptRepository.save(MessageReceipt.of("bb8323b9-1023-4046-b620-63c4f9120b62", ServiceIdentifier.DPV));
-        messageReceiptRepository.save(MessageReceipt.of("bb8323b9-1023-4046-b620-63c4f9120b63", ServiceIdentifier.DPV));
+//        messageReceiptRepository.save(MessageReceipt.of("bb8323b9-1023-4046-b620-63c4f9120b62",
+//                "123", "foo", ServiceIdentifier.EDU));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)